src/logic/operate.test.js

```diff
@@ -0,0 +1,8 @@
+import operate from './operate';
+
+describe('operate', () => {
+  it('should return the sum of two numbers when passed "+" as the operation', () => {
+    const result = operate('5', '3', '+');
+    expect(result).toEqual('8');
+  });
+});
```

Comment:
- [ ] Good job so far 👍  I noticed that you only write unit tests for the files operate.js and calculate.js using Jest . But according to project requirements, you should create unit tests for all React components by using Jest snapshots to test the components and React Testing Library to simulate user interaction. Please make sure that your full tests suite using Jest and all tests should pass 👍